test/src/org/example/cucumber/StepDefinitions.java

```diff
@@ -0,0 +1,32 @@
+package org.example.cucumber;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.junit.Assert;
+
+import org.example.Fridge;
+
+public class StepDefinitions {
+    private Fridge fridge;
+
+    private int remainingBottles;
+
+    @Given("I have {int} water bottles")
+    public void i_have_water_bottles(int int1) {
```

Comment:
Could it be more readable if the arguments were called `bottles` or `numberOfBottles` instead of `int1` etc.?

#namingthings...

Reply:
Done this!